src/server/scripts/Northrend/zone_sholazar_basin.cpp

```diff
@@ -305,6 +317,8 @@ class npc_still_at_it_trigger : public CreatureScript
 
         npc_still_at_it_triggerAI(Creature* pCreature) : NullCreatureAI(pCreature) {}
 
+        Creature* GetManus() {return ObjectAccessor::GetCreature(*me, thunderbrewGUID);}
```

Comment:
is there ever a risk of this returning null?

Reply:
AFAIK, no https://github.com/azerothcore/azerothcore-wotlk/blob/master/src/server/scripts/Northrend/zone_sholazar_basin.cpp#L491, creature cannot die and cannot engage into any type of combat, doesn't despawn and GUID is obtained only by speaking to it (which is the only way the script begins in first place).

If I wanted to add a check, would it be something like this:
```c++
void GetManus()
{
        if (Creature* cr= ObjectAccessor::GetCreature(*me, thunderbrewGUID))
        {
                return cr;
        }
        else
        {
                Reset();
        }
}